### PR TITLE
added props onDragStart and onDragEnd to control when is dragging

### PR DESCRIPTION
**Pull request by jordisantamaria**
To add what discussed in https://github.com/jacobp100/react-twentytwenty/issues/16

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -92,15 +92,28 @@ export default class TwentyTwenty extends Component {
...
     document.addEventListener('touchend', this.endDrag);
+
+    if (this.props.onDragStart) {
+      this.props.onDragStart();
+    }
   }
 
-  endDrag() {
+  endDrag(e) {
+    if (e) {
```
Comment (jacobp100) [inline src/index.js:102]: What use-case does this solve?

**Comment by jacobp100** [comment]
Looks good! Just had a question about the `stopPropagation`

**Comment by jordisantamaria** [comment]
I'm not sure It was need, but I added it on the project I was working with, to avoid propagate click event of the element when drag ends.

**Comment by jacobp100** [comment]
Could we leave that part out? I'm just concerned it might cause bugs. Should be good to go after that!

**Comment by jordisantamaria** [comment]
Sure, you can leave that part yep.

________________________________
De: Jacob Parker <notifications@github.com>
Enviado: jueves, 1 de noviembre de 2018 10:02:17
Para: jacobp100/react-twentytwenty
Cc: Jordi Santamaria Portoles; Author
Asunto: Re: [jacobp100/react-twentytwenty] added props onDragStart and onDragEnd to control when is dragging (#17)


Could we leave that part out? I'm just concerned it might cause bugs. Should be good to go after that!

**Comment by jacobp100** [comment]
I'll only be able to update this when I have free time. If you needed this more urgently, I'll be able to click the merge button with the propagation part removed and a rebuild. 😀

**Comment by jordisantamaria** [comment]
Nah, its not urgent don't worry

________________________________
De: Jacob Parker <notifications@github.com>
Enviado: jueves, 1 de noviembre de 2018 15:49:01
Para: jacobp100/react-twentytwenty
Cc: Jordi Santamaria Portoles; Author
Asunto: Re: [jacobp100/react-twentytwenty] added props onDragStart and onDragEnd to control when is dragging (#17)


I'll only be able to update this when I have free time. If you needed this more urgently, I'll be able to click the merge button with the propagation part removed and a rebuild. 😀

**Pull request closed without merging**